app/models/claims/claim_window.rb

```diff
@@ -0,0 +1,75 @@
+# == Schema Information
+#
+# Table name: claim_windows
+#
+#  id               :uuid             not null, primary key
+#  discarded_at     :date
+#  ends_on          :date
+#  starts_on        :date
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  academic_year_id :uuid             not null
+#
+# Indexes
+#
+#  index_claim_windows_on_academic_year_id  (academic_year_id)
+#  index_claim_windows_on_discarded_at      (discarded_at)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (academic_year_id => academic_years.id)
+#
+class Claims::ClaimWindow < ApplicationRecord
+  include Discard::Model
+
+  default_scope -> { kept }
+
+  belongs_to :academic_year, optional: true
```

Comment:
This doesn't seem like it should be `optional: true`

Reply:
This is a bit of an annoyance. Yes, it shouldn't be optional.

Although, if this is required, if the `starts_on` value is missing, the missing academic year error is exposed in the form.

![CleanShot 2024-07-18 at 15 19 15](https://github.com/user-attachments/assets/195eb3bf-c285-4166-9d96-82f4bffe4e8c)

The proper solution is to extract these validations out into a FormObject, but for the sake of meeting delivery constraints, this is a worthwhile workaround.